src/xcode/ENA/ENA/Source/Views/ExposureSubmission/DynamicAcknowledgementCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+@IBDesignable
+class DynamicAcknowledgementCell: UITableViewCell {
+
```

Comment:
MARKs are missing, pleas add and reorder

Reply:
For 40 lines of code, including white space?

Reply:
MARKs will help if the file gets lager over time, please add them